icu4c/source/common/dictbe.cpp

```diff
@@ -867,6 +871,7 @@ KhmerBreakEngine::divideUpDictionaryRange( UText *text,
                                                 int32_t rangeStart,
                                                 int32_t rangeEnd,
                                                 UVector32 &foundBreaks,
+                                                UBool isPhraseBreaking,
```

Comment:
If you got compiler error, change to 
                             UBool /* isPhraseBreaking */,